witnet.toml

```
@@ -91,6 +91,8 @@ genesis_path = ".witnet/config/genesis_block.json"
 # the percentage of the block rewards that will be assigned to `mint_external_address` (50% by default)
 #mint_external_address = "twit1jqgf4rxjrgas3kdhj3t4cr3mg3n33m8zw0aglr"
 #mint_external_percentage = 50
+# Set a minimum fee you require before your node includes a value transfer transaction into a block
+minimum_vtt_fee = 1
```

This will set this value to 1 for people who use the default witnet.toml
Comment this line to use the default value, 0.

```suggestion
#minimum_vtt_fee = 1
```

---

I set it to 1 because (currently) most node operators are not quite the type of people that are familiar enough with node operation to modify that type of parameters (even though it would still make sense for them to request a minimum fee of 1 nanoWit). However, if you still think it is better to set it to the default value of 0, I'm fine with that.

---

Well, it's easy to approve PRs that add extra features and don't change existing ones, but if some feature changes the default behavior of a node then those changes need to be carefully discussed.

I would prefer to keep the value of 0 because I don't see transactions with 0 fee as a big problem, but let's wait for a third opinion.

---

I agree, that's why I'm taking the time to discuss this. While it is not a breaking change in the codebase, the impact on the economics of the network are definitely not neglible. I'm not in a rush to get this merged, so yes, let's wait for more opinions.   😉

---

I agree that at least 1 nanowit is need for a miner to earn a really minimum profit by include the transaction in the block